core/Makefile

```diff
@@ -83,7 +83,7 @@ endif
 # user requests a static build.
 ALWAYS_DYN_LIBS := -lpthread -ldl
 # These libraries are not supported by pkg-config.
-ALWAYS_LIBS := -lpcap -lgflags
+ALWAYS_LIBS := -lpcap -lgflags -lnuma
 # If pkg-config is available, we just need a list of the dependecies.
```

Comment:
@ddiproietto  Could you confirm this change are appropriate?

Reply:
The change makes sense, thanks